app/dataaccess/model/RecruitmentStage.py

```diff
@@ -0,0 +1,36 @@
+from enum import Enum
+
+from bson import ObjectId
+
+from .BaseObject import BaseObject
+from .RecruitmentInstruction import StageType
+
+
+class RecruitmentStageStatus(Enum):
+    CREATED = 1
+    IN_PROGRESS = 2
+    DONE = 3
+
+
+class RecruitmentStage(BaseObject):
+    def __init__(
+        self,
+        _id: str | ObjectId,
+        recruitment_id: str | ObjectId,
+        identifier: int,
```

Comment:
za co odpowiada to pole?

Reply:
to jest de facto indeks z tej petli, ktora dodaje typ i priorytet do rsm agentow plus dodaje go do jid tego agenta (prosciej mi bylo w logach ogarnac co jest czym jak mialem jid rsm agentow postaci <recruitment_id>_<identifier>)